environments.json

```diff
@@ -5170,10 +5190,45 @@
   "node19_0": {
     "full": "Node.js",
     "family": "Node.js",
-    "short": "Node &gt;=19",
+    "short": "Node &gt;=19 &lt;19.2",
     "platformtype": "engine",
     "equals": "chrome107",
     "release": "2022-10-17",
+    "obsolete": true,
+    "test_suites": [
+      "es5",
+      "es6",
+      "es2016plus",
+      "esnext",
+      "esintl",
+      "non-standard"
+    ]
+  },
+  "node19_2": {
+    "full": "Node.js",
+    "family": "Node.js",
+    "short": "Node &gt;=19.2 &lt;20",
+    "platformtype": "engine",
+    "equals": "chrome108",
+    "release": "2022-11-29",
```

Comment:
Node.js 19 just reaches EOL this month. I suggest we wait for a while before marking it as obsolete.